backends-velox/src/main/scala/org/apache/spark/sql/execution/ColumnarCachedBatchSerializer.scala

```diff
@@ -233,12 +233,13 @@ class ColumnarCachedBatchSerializer extends CachedBatchSerializer with SQLConfHe
       it =>
         val nmm = NativeMemoryManagers
           .contextInstance("ColumnarCachedBatchSerializer read")
+        val execCtx = ExecutionCtxs.contextInstance()
         val schema = SparkArrowUtil.toArrowSchema(localSchema, timezoneId)
         val arrowAlloc = ArrowBufferAllocators.contextInstance()
         val cSchema = ArrowSchema.allocateNew(arrowAlloc)
         ArrowAbiUtil.exportSchema(arrowAlloc, schema, cSchema)
         val deserializerHandle = ColumnarBatchSerializerJniWrapper
-          .create()
+          .create(execCtx)
```

Comment:
Is this change actually the one fixes the issue? Which stores a local variable of `ExecutionCtxs.contextInstance()`?